core/src/test/java/org/infinispan/eviction/impl/EvictionWithPassivationTest.java

```diff
@@ -9,34 +9,53 @@
 
 import org.infinispan.Cache;
 import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.StorageType;
 import org.infinispan.manager.EmbeddedCacheManager;
 import org.infinispan.persistence.dummy.DummyInMemoryStoreConfigurationBuilder;
 import org.infinispan.test.SingleCacheManagerTest;
 import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.testng.annotations.Factory;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "eviction.EvictionWithPassivationTest")
 public class EvictionWithPassivationTest extends SingleCacheManagerTest {
 
    private static final String CACHE_NAME = "testCache";
+   private final int EVICTION_MAX_ENTRIES = 2;
+   private StorageType storage;
 
    public EvictionWithPassivationTest() {
       // Cleanup needs to be after method, else LIRS can cause failures due to it not caching values due to hot
       // size being equal to full container size
       cleanup = CleanupPhase.AFTER_METHOD;
    }
 
-   private final int EVICTION_MAX_ENTRIES = 2;
+   @Factory
+   public Object[] factory() {
+      return new Object[] {
+            new EvictionWithPassivationTest().withStorage(StorageType.BINARY),
+            new EvictionWithPassivationTest().withStorage(StorageType.OBJECT),
```

Comment:
Is `BINARY` any different from `OBJECT` when it comes to eviction/passivation?

Reply:
It shouldn't be, but no reason not to just run all 3 :)